### PR TITLE
Make tests faster

### PR DESCRIPTION
**Pull request by freesig**

### CODE REVIEW

File: nix/pkgs/core.nix [hunk 1]
```diff
@@ -17,14 +17,14 @@ rec {
     export CARGO_BUILD_JOBS=4
 
     # ensure plain build works
-    cargo build --no-default-features --manifest-path=crates/holochain/Cargo.toml
+    # cargo build --no-default-features --manifest-path=crates/holochain/Cargo.toml
```
Comment (steveeJ) [inline nix/pkgs/core.nix:20]: I'm fine removing this as it doesn't fully simulate the Nix builds anyway. Please remove this line and its comment altogether.

Comment (steveeJ) [inline nix/pkgs/core.nix:20]: I'm fine removing this as it doesn't fully simulate the Nix builds anyway. Please remove this line and its comment altogether.

File: nix/pkgs/core.nix [hunk 2]
```diff
@@ -17,14 +17,14 @@ rec {
...
 
     # alas, we cannot specify --features in the virtual workspace
     cargo test warm_wasm_tests --manifest-path=crates/holochain/Cargo.toml --features slow_tests,build_wasms
```
Comment (steveeJ) [inline nix/pkgs/core.nix:23]: This is now included in the next invocation as the `warm_wasm_tests` merely acts as a filter, and the next invocation runs the identical feature set without any filter. I.e. this line can be removed now

**Review by steveeJ — COMMENTED** [review]
Thanks, saving time on CI is great :partying_face: I left some comments to further improve this change.

**Review by thedavidmeister — APPROVED** [review]

**Review by steveeJ — APPROVED** [review]
With the most recent changes we're down to 17 minutes per CI run which is even faster than before we decreased concurrency :speedboat: I think that we could likely go higher than 8 but let's leave the upper limit search for another time and get this improvement in.

**Pull request merged**